NebulaClient/MultiplayerClientSession.cs

```diff
@@ -73,6 +79,23 @@ private void ConnectInternal()
             LocalPlayer.SetNetworkProvider(this);
         }
 
+        public void DisplayPingIndicator()
+        {
+            if (pingIndicator == null)
+            {
+                GameObject targetObject = GameObject.Find("label");
+                pingIndicator = GameObject.Instantiate(targetObject, targetObject.transform.parent.parent).GetComponent<Text>();
```

Comment:
Can this be a more specific UI gameobject? I think UIRoot or UIGame has members for important layers.

Reply:
`transform.parent.parent` is `UIGame` object.
 But I can also change it to:
 ```cs
 GameObject.Instantiate(targetObject, GameObject.Find("in game").transform)
 ```
 But it would be same.

Reply:
Or you can use `UIRoot.instance.uiGame` then?

Reply:
The reasoning is that if it is removed from the game, you get a compile error.

You don't get it if the unity scene is reorganized

Reply:
true,

Reply:
fixed :)